setup.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-#
-# Picard, the next-generation MusicBrainz tagger
-#
+"""Picard, the next-generation MusicBrainz tagger"""
+
```

Comment:
The header is written by [scripts/tools/fix-header.py](https://github.com/metabrainz/picard/blob/master/scripts/tools/fix-header.py). If we want to remove the coding part should be done there and the script rerun to update the whole project.

Also "Picard, the next-generation MusicBrainz tagger" should stay as a comment. This belongs to the copyright lines following it. Some files in the repo do have an explicit doc comment that is separate from the copyright comment. I'm not sure whether a doc comment for setup.py is any use, but "Picard, the next-generation MusicBrainz tagger" doesn't appear to me of being of much use.

Reply:
I've only left the version change in this PR then